### DISCUSSION TITLE
Project Beta: Allow fine grained access control

### DISCUSSION THREAD

**Discussion by edwdev** [community/community#5415] [Projects and Issues]
We are are provider with multiple customers, that have access to various repo's in our organisation. Both as ORG members and outside collaborators.

It didn't seem like this was possible (please correct me if I missed it). It would be great, if it was possible to simply select a Project (Beta), and choose exactly what users/groups/collaborators should have access to the project (including read/write) per user/group. 

It also doesn't seem possible at the moment to change a Beta project from private <> Public.

**Comment by luxaritas** [comment]
> It also doesn't seem possible at the moment to change a Beta project from private <> Public
Note this has been launched now (see github/feedback#5681)

That said, more fine-grained access controls would be good!

**Comment by luxaritas** [comment]
I take that back - it looks like this has actually been shipped. Under the project settings, I see a "manage access" tab that allows for read/write/admin just like a repo does!

**Reply by scherii** [reply]
It seems that this doesn't apply to Outside Collaborators yet:
https://github.com/github/feedback/discussions/7562
